### PR TITLE
[IOTDB-6236] Pipe: CountPointProcessor example

### PR DESCRIPTION
**Pull request by yschengzi**
Simple PipeProcessor plugin:
Main function: In standalone case, it can count the total number of IoTDB points in real time and write them into a target sequence (specified by the KV parameter of the Processor).

简易 PipeProcessor 插件：
主要功能：单机情况下，可以实时统计 IoTDB 的总点数并写成目标序列（由 Processor 的 KV 参数指定）

使用方法：
>IoTDB: create pipeplugin c as 'org.apache.iotdb.CountPointProcessor' using URI 'file:///local/count-point-processor-example-1.3.0-SNAPSHOT.jar'
>IoTDB: create pipe p with processor( 'processor' = 'c', 'aggregate-series'='root.sg.res.count', ) with connector ( 'connector'='iotdb-thrift-connector', 'connector.ip'='127.0.0.1', 'connector.port'='6668',)

### CODE REVIEW

File: example/count-point-processor/pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>pipe-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-server</artifactId>
+            <version>${project.version}</version>
```
Comment (SteveYurongSu) [inline example/count-point-processor/pom.xml:45]: scope provided

File: example/pom.xml [hunk 2]
```diff
@@ -47,6 +47,7 @@
         <module>rest-java-example</module>
         <module>flink-sql</module>
         <module>schema</module>
+        <module>count-point-processor</module>
```
Comment (SteveYurongSu) [inline example/pom.xml:50]:
```suggestion
        <module>pipe-count-point-processor</module>
```

Comment (SteveYurongSu) [reply example/pom.xml:50]: also change name from opc-ua-sink to pipe-opc-ua-sink

File: example/count-point-processor/pom.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.iotdb</groupId>
+        <artifactId>iotdb-examples</artifactId>
+        <version>1.3.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>count-point-processor-example</artifactId>
+    <name>IoTDB: Example: CountPointProcessor</name>
```
Comment (SteveYurongSu) [inline example/count-point-processor/pom.xml:30]:
```suggestion
    <artifactId>pipe-count-point-processor-example</artifactId>
    <name>IoTDB: Example: Pipe: CountPointProcessor</name>
```

Comment (yschengzi) [reply example/count-point-processor/pom.xml]: done.

File: example/pom.xml (hunk 2 above)
Comment (yschengzi) [reply example/pom.xml]: done.

File: example/count-point-processor/pom.xml (hunk 1 above)
Comment (yschengzi) [reply example/count-point-processor/pom.xml]: done.

**Review by SteveYurongSu — APPROVED** [review]

**Pull request merged**